src/otx/algo/instance_segmentation/mmdet/models/dense_heads/rtmdet_ins_head.py

```diff
@@ -0,0 +1,1225 @@
+"""RTMDet-Ins head."""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) OpenMMLab. All rights reserved.
+from __future__ import annotations
+
+import copy
+import math
+from typing import TYPE_CHECKING
+
+import torch
+import torch.nn.functional
+from mmengine.structures import InstanceData
```

Comment:
we can replace this to `from otx.algo.utils.mmengine_utils import InstanceData`

Reply:
I will clean these up in inst-seg data structure decoupling PR.